### ISSUE TITLE
stubbed eth1data votes for interop testing

### ISSUE THREAD

**Issue by djrtwo** [ethereum/eth2.0-pm#62]
# stubbed eth1data votes for interop testing

Considering most of our initial interop tests will not have an associated eth1 chain to come to consensus upon via the eth1data vote mechanism, it is worth coming to a quick standard on how clients should stub this vote. The goal here is to actually _not_ successfully come to consensus on a new eth1data value because they would then require deposit processing.

We may want to come up with a more clever mechanism that can actually handle deposits in the future, but for now we should just operate in such a way that disables deposits.

A proposer should fill their `block.body.eth1_data` with the following:

```python
Eth1Data(
    deposit_root=hash(current_epoch),
    deposit_count=slot,
    block_hash=hash(hash(current_epoch)),
)
```

## Alternative

An alternative to test that the eth1data mechanism actually updates is to use a mechanism that agrees upon `deposit_root` and `block_hash` and always keeps `deposit_count` at `state.eth1_deposit_index` so that the consensus does not force deposits even as eth1data is updated. 

We could use the following for this:

```python
epochs_per_period = SLOTS_PER_ETH1_VOTING_PERIOD // SLOTS_PER_EPOCH
voting_period = current_epoch // epochs_per_period
Eth1Data(
    deposit_root=hash(voting_period),
    deposit_count=state.eth1_deposit_index,
    block_hash=hash(hash(voting_period)),
)

**Comment by ralexstokes** [comment]
the primary goal here is to keep it simple, but in the interest of being able to track distinct inputs, perhaps we combine both?

```python
slot_in_voting_period = current_slot % SLOTS_PER_ETH1_VOTING_PERIOD
Eth1Data(
    deposit_root=hash(current_epoch + slot_in_voting_period),
    deposit_count=state.eth1_deposit_index,
    block_hash=hash(hash(current_epoch + slot_in_voting_period),
)
```

**Comment by djrtwo** [comment]
I had an error in the second. I intended for the `voting_period` to be stable so that consensus on eth1 data can actually occur. updating

**Comment by ralexstokes** [comment]
yeah, the updated alternative seems good

**Comment by djrtwo** [comment]
Leaning in favor of the "Alternative" at this point. any thoughts out there?

**Comment by paulhauner** [comment]
I also favor the "Alternative" solution

**Comment by djrtwo** [comment]
closed via #72
